examples/ul2/src/super_glue/metrics.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2022 MosaicML Examples authors
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Optional
+
+import torch
+from torchmetrics import Metric
+
+
+class ExactMatch(Metric):
+    is_differentiable = False
+    higher_is_better = True
+    full_state_update = False
```

Comment:
This class should probably be in Composer (though again, okay for now)

Reply:
I'll also leave this as a future TODO for now, but noted.